src/Microsoft.Extensions.Testing.Abstractions/project.json

```diff
@@ -14,6 +14,7 @@
     "Newtonsoft.Json": "7.0.1",
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "Microsoft.Extensions.Logging.Abstractions": "1.0.0-rc2-20581",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-rc2-20581",
```

Comment:
We need to recompile `dotnet-test-xunit against this build of Testing.Abstractions before we can remove this. Not having this dependency causes loader exceptions. I have a similar change for Logging.Abstractions https://github.com/dotnet/cli/pull/2732/files and I'll address those both once these changes go in.